.meteor/packages

```diff
@@ -4,38 +4,38 @@
 # but you can also edit it by hand.
 
 rocketchat:cors
-accounts-facebook@1.1.0
-accounts-github@1.2.0
-accounts-google@1.1.0
-accounts-meteor-developer@1.2.0
+accounts-facebook@1.1.1
+accounts-github@1.2.1
+accounts-google@1.1.1
+accounts-meteor-developer@1.2.1
 accounts-password@1.3.4
-accounts-twitter@1.2.0
+accounts-twitter@1.2.1
 blaze-html-templates
-check@1.2.4
+check@1.2.5
 coffeescript@1.11.1_4
-ddp-rate-limiter@1.0.6
+ddp-rate-limiter@1.0.7
 ecmascript@0.6.3
 ejson@1.0.13
 email@1.1.18
 fastclick@1.0.13
-http@1.2.11
+http@1.2.12
 jquery@1.11.10
 less@2.7.9
 logging@1.1.17
 meteor-base@1.0.4
 mobile-experience@1.0.4
-mongo@1.1.15
+mongo@1.1.16
```

Comment:
Can you undo changes to this file?  Like mentioned in the other comment, ideally we'd handle these types of updates in a seperate PR.  This should only about adding drupal.  Thx :+1: